app/src/test/java/io/hstream/testing/BasicTest.java

```diff
@@ -197,7 +199,7 @@ void testMultiProducer() throws Exception {
   @Disabled("restart may cause test fail, disable")
   @Test
   @Timeout(60)
-  void testReconsumeAfterRestartServer() throws Exception {
+  void testReconsumeAllDataFromSameSubscriptionAfterServerRestart() throws Exception {
```

Comment:
Actually, they are not the same subscription... I do not know how to describe it. Maybe some comments are needed.

Reply:
You are right, the test actually tests the ability to repeatedly consume from a persistent stream, I removed it from the test set because it is not related to the server.